scripts/build.py

```diff
@@ -180,10 +183,10 @@ def get_cache_slug():
     EXTENSIONS_SRC_DIR = os.path.join('extensions', '')
     EXTENSIONS_OUT_DIR = os.path.join(BUILD_DIR, 'extensions', '')
     # Certain files' syntax become incorrect after minification and hence
-    # they are ignored.
-    IGNORE_PATHS = [os.path.join('extensions', 'interactions',
-                                 'LogicProof', 'static', 'js')]
-    build_files(EXTENSIONS_SRC_DIR, EXTENSIONS_OUT_DIR, IGNORE_PATHS)
+    # they are copied without minification.
+    COPY_WITHOUT_MINIFICATION = [os.path.join('extensions', 'interactions',
+                                              'LogicProof', 'static', 'js')]
+    build_files(EXTENSIONS_SRC_DIR, EXTENSIONS_OUT_DIR, COPY_WITHOUT_MINIFICATION)
```

Comment:
It's still giving errors. I think it occurs in all the places like 'boolean' which are js keywords.